### PR TITLE
Feature/most popular tags

### PR DESCRIPTION
**Pull request by logston**

### CODE REVIEW

**Comment by ElliotJH** [comment]
LGTM 👍

File: plugins/aggregations.py [hunk 1]
```diff
@@ -30,6 +30,8 @@ def _handle_content_object_init(obj):
                 _categories[category.slug]['count'] = count + 1
                 _categories[category.slug]['latest'] = latest
                 _categories[category.slug]['name'] = category.name
+
+        speaker = getattr(obj, 'author')
```
Comment (zerok) [inline plugins/aggregations.py]: The author is not optional, right? Perhaps we could use the opportunity to let the script die here if no author is provided 😃

**Comment by zerok** [comment]
Looks good 😃

File: plugins/aggregations.py (hunk 1 above)
Comment (logston) [inline plugins/aggregations.py]: Good call.

**Pull request merged**